### PR TITLE
Nic mappings

### PR DESCRIPTION
**Pull request by machacekondra**
Signed-off-by: Ondra Machacek <omachace@redhat.com>

### CODE REVIEW

**Review by pkliczewski — APPROVED** [review]

File: pkg/providers/ovirt/mapper/mapper.go [hunk 1]
```diff
@@ -176,6 +182,75 @@ func (o *OvirtMapper) MapDisks() map[string]cdiv1.DataVolume {
...
+	var kubevirtNics []kubevirtv1.Interface
+	nics, _ := o.vm.Nics()
+	for _, nic := range nics.Slice() {
+		// This network interface don't have any network specified.
+		if _, ok := nic.VnicProfile(); !ok {
+			continue
+		}
+
+		kubevirtNic := kubevirtv1.Interface{}
+		kubevirtNic.Name, _ = nic.Name()
```
Comment (jakub-dzon) [inline pkg/providers/ovirt/mapper/mapper.go:195]: Aren't there any restrictions on the name format - like with the vm name? Shouldn't it be normalized?

File: pkg/providers/ovirt/mapper/mapper.go [hunk 2]
```diff
@@ -176,6 +182,75 @@ func (o *OvirtMapper) MapDisks() map[string]cdiv1.DataVolume {
...
+	nics, _ := o.vm.Nics()
+	for _, nic := range nics.Slice() {
+		// This network interface don't have any network specified.
+		nicProfile, ok := nic.VnicProfile()
+		if !ok {
+			continue
+		}
+
+		kubevirtNet := o.getNetworkForNic(nicProfile)
+		kubevirtNet.Name, _ = nic.Name()
```
Comment (jakub-dzon) [inline pkg/providers/ovirt/mapper/mapper.go:219]: Same question as with the interface, as they have to match.

**Comment by jakub-dzon** [comment]
Also - the build has failed.

File: pkg/providers/ovirt/mapper/mapper.go (hunk 1 above)
Comment (machacekondra) [reply pkg/providers/ovirt/mapper/mapper.go:195]: I am not sure, I will check.

Comment (machacekondra) [reply pkg/providers/ovirt/mapper/mapper.go:195]: oVirt backend already implements the validation and only nic with alphanumeric characters and '-', '_' can be created. So I think we don't need the validation here.

Comment (jakub-dzon) [reply pkg/providers/ovirt/mapper/mapper.go:195]: :+1:

**Review by jakub-dzon — APPROVED** [review]

**Pull request merged**